apps/prairielearn/src/pages/publicQuestionPreview/publicQuestionPreview.ts

```diff
@@ -43,7 +43,7 @@ async function setLocals(req, res) {
 
 router.post(
   '/',
-  asyncHandler(async (req, res) => {
+  typedAsyncHandler<'course-question'>(async (req, res) => {
```

Comment:
This makes me a little nervous. The public question preview pages live in their own little world independent of the rest of our usual middleware. Is it safe to mark these routes like this?

Reply:
It still lives within a course, regardless of it is public or not. My thinking here is that if the typing is incorrect, it's going to blow up for someone, and at that point we can improve the typing further.